modules/core/04-channel/v2/types/packet_test.go

```diff
@@ -0,0 +1,137 @@
+package types
```

Comment:
I wanted to use the constants in `ibctesting` here but got in this import cycle:

```
ackage github.com/cosmos/ibc-go/v9/modules/core/04-channel/v2/types
	imports github.com/cosmos/ibc-go/v9/testing
	imports github.com/cosmos/ibc-go/v9/modules/core/keeper
	imports github.com/cosmos/ibc-go/v9/modules/core/types
	imports github.com/cosmos/ibc-go/v9/modules/core/04-channel/v2/types: import cycle not allowed in test
```

Reply:
`package types_test` should do the trick!

Reply:
it does, thanks all!